### PR TITLE
Fix and disable failing BBE tests

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
Fix or disable failing BBE test cases due to xml syntax change.
And update xml BBEs

This PR does not update xml-functions BBE. It will be update it with a second PR.

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/xml-access/xml_access.bal [hunk 1]
```diff
@@ -14,25 +14,30 @@ public function main() {
...
 
-    // Accessing a non-existing child will return `nil`.
-    io:println(bookXML.ISBN.code);
-    io:println(bookXML["ISBN"]["code"]);
-
-    // The result of the above field-based or index-based access is another XML. 
-    // Any XML function can be invoked on top of the resulting XML.
-    io:println(bookXML.author.fname.getTextValue());
-    io:println(bookXML["author"]["fname"].getTextValue());
+    // Accessing a non-existing child will return empty xml sequence.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.bal]:
```suggestion
    // Accessing a non-existing child will return am empty `xml` sequence.
```
or
```suggestion
    // Accessing a non-existing child will return an empty XML sequence.
```

File: examples/xml-access/xml_access.bal [hunk 2]
```diff
@@ -14,25 +14,30 @@ public function main() {
...
+    io:println(bookXML/<ISBN>/<code>);
 
     // You can also retrieve attributes of the resulting child XML.
-    io:println(bookXML.author.fname@["title"]);
-    io:println(bookXML["author"]["fname"]@["title"]);
+    io:println(bookXML/<author>/<fname>.title);
+
+    // You can match descendent elements using following steping access syntax.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.bal]:
```suggestion
    // You can match descendant elements using the following stepping access syntax.
```

File: examples/xml-access/xml_access.bal [hunk 3]
```diff
@@ -14,25 +14,30 @@ public function main() {
...
+    io:println(bookXML/**/<fname>);
+
+    // Select all children using bellow syntax.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.bal]: `// Use the following syntax to select all the child elements`
or
`// Use the following syntax to select all the children`

File: examples/xml-access/xml_access.bal [hunk 4]
```diff
@@ -14,25 +14,30 @@ public function main() {
...
+    io:println(bookXML/*);
+
+    // And select all children elements using bellow syntax.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.bal]: What's the difference between the above selection and the below one?

File: examples/xml-access/xml_access.bal [hunk 5]
```diff
@@ -14,25 +14,30 @@ public function main() {
...
+    io:println(bookXML/<*>);
+
+    // Select all children belong to specific namespace.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.bal]: `// Select all the children belonging to a specific namespace.

File: examples/xml-access/xml_access.bal [hunk 6]
```diff
@@ -14,25 +14,30 @@ public function main() {
...
+    xmlns "http://ballerina.com/a" as bar;
+    io:println(bookXML/<bar:*>/*);
 
-    // You can access the elements by their namespace-qualified (fully-qualified) name.
-    io:println(bookXML["{http://ballerina.com/a}year"].getTextValue());
-    xmlns "http://ballerina.com/a" as foo;
-    io:println(bookXML[foo:year].getTextValue());
+    xml seq = bookXML/*;
+    // You can filter a xml sequence using xml filter expressions.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.bal]: ` // XML sequences can be filtered using XML filter expressions.`

File: examples/xml-access/xml_access.description [hunk 7]
```diff
@@ -1,2 +1,2 @@
 // XML elements can contain child XML items. Ballerina allows you to access these
-// child elements conveniently using field-access and index-access expressions.
+// child elements conveniently using xml step expressions and xml filter expressions.
```
Comment (pubudu91) [inline examples/xml-access/xml_access.description]: xml -> XML. Use `xml` if referring to the XML type in Ballerina. If referring to XML in general, use XML (all caps, not highlighted as code)

File: examples/xml-attributes/xml_attributes.bal [hunk 8]
```diff
@@ -1,41 +1,33 @@
 import ballerina/io;
+import ballerina/lang.'xml as xmllib;
```
Comment (pubudu91) [inline examples/xml-attributes/xml_attributes.bal]: Do we need to use an alias here?

File: examples/xml-attributes/xml_attributes.bal [hunk 9]
```diff
@@ -1,41 +1,33 @@
...
-    string? s = x1@["{http://ballerina.com/aa}status"];
-    io:println(s);
-
-    // An attribute that is not bound to a namespace can be accessed using the string representation of the name.
-    io:println(x1@["count"]);
-
-    // Accesses an attribute using a dynamic name.
-    string attributeName = "count";
-    io:println(x1@[attributeName]);
+    // Attribute access expressions are lax typed.
```
Comment (pubudu91) [inline examples/xml-attributes/xml_attributes.bal]: We don't have any documentation except the spec which explains what lax typing is right? If that's the case, we might either have to explain briefly what lax typing is or just explain what's going on here, without referring to lax typing.

File: examples/xml-attributes/xml_attributes.bal [hunk 10]
```diff
@@ -1,41 +1,33 @@
...
+    string|error count = x1.count;
+    io:println(count);
 
-    // Updates an attribute.
-    x1@[ns0:status] = "Not Available";
-    io:println(x1@[ns0:status]);
+    // Returns an error when accessing non existing attribute.
```
Comment (pubudu91) [inline examples/xml-attributes/xml_attributes.bal]: `// Accessing a non-existent attribute will return an error.`

File: examples/xml-attributes/xml_attributes.bal [hunk 11]
```diff
@@ -1,41 +1,33 @@
...
+    string|error count2 = x1.count2;
+    io:println(count2 is error);
 
-    // It is possible to get all the attributes at once. However, this cannot be assigned to any variable.
-    io:println(x1@);
-
-    // The`x1@` syntax can be used to get all the attributes of a singleton XML element as a map.
-    map<string>? attributeMap = x1@;
+    // It is possible to get all the attributes of a xml element.
```
Comment (pubudu91) [inline examples/xml-attributes/xml_attributes.bal]: xml -> XML

File: examples/xml-attributes/xml_attributes.bal [hunk 12]
```diff
@@ -1,41 +1,33 @@
...
-    // However, if we apply the `@` operator to a XML sequence, it will return NIL because only XML elements have attributes.
-    xml x2 = xml `<Person name="James"/>`;
-    xml x3 = x1 + x2;
-    map<string>? attributeMapOfSeq = x3@;
-    io:println(attributeMapOfSeq);
+    // An attribute can also be accessed using the string representation of the qualified name from the attribute map.
+    string? s = attributeMap["{http://ballerina.com/aa}status"];
+    io:println(s);
+
+    // Updates an attribute by updating the attribute map of a xml element.
```
Comment (pubudu91) [inline examples/xml-attributes/xml_attributes.bal]: a xml -> an XML

File: examples/xml-access/xml_access.bal (hunk 3 above)
Comment (rdhananjaya) [reply examples/xml-access/xml_access.bal]: children will include non elements (pi, comment, text)

File: examples/xml-attributes/xml_attributes.description [hunk 13]
```diff
@@ -1,4 +1,7 @@
 // XML elements may have any number of attributes and any number
 // of namespace declarations that apply for that element. 
 // In Ballerina, both of these types are treated the same. 
-// Attributes are accessed from an XML sequence using the `@` postfix operator.
+// XML attribute access in ballerina is lax typed similar
+// to `json`, we can use field access expression (`.`) and
+// optional field access expressions (``?.`) to access teh
+// field of a xml value.
```
Comment (pubudu91) [inline examples/xml-attributes/xml_attributes.description]: The above needs to be slightly rephrased. Again, if we don't have an example for lax typing, users may be confused as to what lax typing is. @MaryamZi thoughts?

File: examples/xml-access/xml_access.bal (hunk 1 above)
Comment (rdhananjaya) [reply examples/xml-access/xml_access.bal]: Will apply to https://github.com/ballerina-platform/ballerina-lang/pull/21826

File: examples/xml-attributes/xml_attributes.description (hunk 13 above)
Comment (MaryamZi) [reply examples/xml-attributes/xml_attributes.description]: There's a typo btw. `teh` -> `the`

**Pull request merged**